ci.sh

```diff
@@ -274,8 +326,10 @@ else
         echo "Start Parsec for stress tests"
         # Change the log level for the stress tests because logging is limited on the
         # CI servers.
-        RUST_LOG=error RUST_BACKTRACE=1 cargo run $FEATURES -- --config $CONFIG_PATH &
-        sleep 5
+        RUST_LOG=error RUST_BACKTRACE=1 cargo run --release $FEATURES -- --config $CONFIG_PATH &
+        wait_for_service
+        # Check that Parsec successfully started and is running
+        pgrep -f target/release/parsec >/dev/null
```

Comment:
Should this be included in `wait_for_service`?

Reply:
Yes!